api/geyser/build.gradle.kts

```diff
@@ -4,6 +4,8 @@ plugins {
 
 dependencies {
     api(projects.api)
+
+    implementation("net.kyori", "adventure-text-serializer-legacy", Versions.adventureVersion)
```

Comment:
We probably can't do this or else Adventure will be in the API which we can't do thanks to shading across platforms.

Reply:
So what do we do?

Reply:
https://github.com/GeyserMC/Geyser/pull/3250/commits/936fed1ded9e8c778e296433a2221c693a9462ba (Camo told me what to do because I am not so smart :) )